nutrition_labels/label_grants.py

```diff
@@ -66,6 +66,7 @@ def get_test_results(file,f1_cutoff = 0,precision_cutoff = 0,recall_cutoff = 0):
     results = lines[7]
     results = re.sub('Test scores: ', '', results)
     results_dict = ast.literal_eval(results)
+    return (results_dict)
```

Comment:
how come you do this? 
It does change how this function works now, so e.g. line 83 `useful_models = [i for i in model_dirs if get_test_results(i,0.8,0.82,0.82)]` wont be working as it should since it expects `get_test_results` to be returning a True or False output not a dictionary.

Reply:
Oh I did it to look at the model results for a doc somewhere and forgot to change it back im so sorry that was a mess up. changed it back